pandas/io/common.py

```diff
@@ -342,9 +342,11 @@ def _get_handle(
     try:
         from s3fs import S3File
 
-        need_text_wrapping = (BytesIO, S3File)
+        need_text_wrapping = (BufferedIOBase, BytesIO, S3File)
```

Comment:
BytesIO is a subclass of BufferedIOBytes. Can BytesIO be removed?

Reply:
👍